core/domain/exp_services.py

```diff
@@ -1926,9 +2120,20 @@ def are_changes_mergeable(exp_id, frontend_version, change_list):
         }
 
         if len(added_state_names) > 0 or len(deleted_state_names) > 0:
-            # Here we will send the changelist, version, latest_version,
-            # and exploration to the admin, so that the conditions
-            # can be reviewed.
+            # Here we will send the changelist, frontend_version,
+            # backend_version and exploration to the admin, so
+            # that the conditions can be reviewed and the changes
+            # can be done to handle those cases.
```

Comment:
Too vague..

"the conditions" - what conditions do you refer to here?

"the changes can be done" - what changes? done where?

"those cases" - which cases?

Reply:
Explained it more clearly now.